TikTokTrainer/Data/DataProcessing.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  DataProcessing.swift
+//  TikTokTrainer
+//
+//  Created by Ankit  on 2/18/21.
+//
+
+import Foundation
+import SwiftUI
+import AVKit
+import Vision
+
+/// Process the PoseNet overlays on the pre-recorded and user-recorded videos and return a score
+struct ScoringFunction {
+    var preRecordedVid: ProcessedVideo?
+    var recordedVid: ProcessedVideo?
+
+    let jointTriples: [(VNRecognizedPointKey, VNRecognizedPointKey, VNRecognizedPointKey)] = [
+        ((VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "left_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "left_hand_joint"))),
+        ((VNRecognizedPointKey(rawValue: "right_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "right_hand_joint"))),
+        ((VNRecognizedPointKey(rawValue: "left_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"))),
+        ((VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "head_joint"))),
+        ((VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_forearm_joint"))),
+        ((VNRecognizedPointKey(rawValue: "head_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_shoulder_1_joint")))
```

Comment:
Can you change these to enums instead of raw string values? Just so that if they change the names the app wont break.